library/Zend/View/Helper/Placeholder/Container/Abstract.php

```diff
@@ -30,6 +30,13 @@
  */
 abstract class Zend_View_Helper_Placeholder_Container_Abstract extends ArrayObject
 {
+    /**
+     * View object
+     * 
```

Comment:
remove trailing whitespace

Reply:
trailing space is still there

Reply:
I fixed here https://github.com/hungtrinh/zf1-future/commit/f269ec7d3b1ec1720b03862cbcef6a52d4e44464